src/Jackett.Common/Definitions/pixelados.yml

```diff
@@ -97,13 +100,36 @@ search:
   fields:
     category:
       selector: category_id
-    title:
+    types:
```

Comment:
the types: and its selector are redundant as you do not reference them anyplace as `.Result.types` below

Reply:
You are right, in the latest version of the code I don't use it, it should be deleted

```suggestion
```

Reply:
You are right, in the latest version of the code I don't use it, it should be deleted

Reply:
Added commit with changes